src/App.tsx

```diff
@@ -15,36 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  length = 'length',
+  alphabet = 'alphabet',
+  default = '',
+}
+
+function getPraparedGoods(goods: string[], sortField: string) {
```

Comment:
```suggestion
function sortGoods(goods: string[], sortField: SortType) {
```
It's not a string, you have enum, so use it. And you don't prepare goods, just sort them